client/my-sites/posts/post-controls.jsx

```diff
@@ -1,226 +1,237 @@
+					className: 'trash is-scary',
 					onClick: this.props.onDelete,
-					icon: 'trash'
+					icon: 'trash',
 				} );
 			} else {
-				availableControls.push( {
-					text: this.translate( 'Trash' ),
-					className: 'post-controls__trash',
+				controls.main.push( {
+					text: translate( 'Trash' ),
+					className: 'trash',
 					onClick: this.props.onTrash,
-					icon: 'trash'
+					icon: 'trash',
 				} );
 			}
 		}
 
-		// --- Extra Controls (behind ... more link) ---
-		if ( ( availableControls.length > 2 && ! this.props.fullWidth ) || ( availableControls.length > 4 && this.props.fullWidth ) ) {
-			moreControlsSpliceIndex = ( ! this.props.fullWidth ) ? 2 : 4;
+		if ( 'publish' === post.status && utils.userCan( 'edit_post', post ) ) {
+			controls.main.push( {
+				text: translate( 'Copy' ),
+				className: 'copy',
+				href: `/post/${ this.props.site.slug }?copy=${ post.ID }`,
+				onClick: this.copy,
+				icon: 'clipboard',
+			} );
+		}
+
+		// More Controls (behind ... more link)
+		if ( ( controls.main.length > 2 && ! this.props.fullWidth ) || ( controls.main.length > 4 && this.props.fullWidth ) ) {
+			const moreControlsSpliceIndex = ( ! this.props.fullWidth ) ? 2 : 4;
 
-			for ( i = moreControlsSpliceIndex; i < availableControls.length; i++ ) {
-				extraControls.push( availableControls[i] );
+			let i;
+			for ( i = moreControlsSpliceIndex; i < controls.main.length; i++ ) {
+				controls.more.push( controls.main[ i ] );
 			}
 
-			availableControls.splice( moreControlsSpliceIndex );
+			controls.main.splice( moreControlsSpliceIndex );
 		}
 
-		// --- Return Controls ---
-		postControlsClass = 'post-controls';
-
-		if ( extraControls.length ) {
-			availableControls.push( {
-				text: this.translate( 'More' ),
-				className: 'post-controls__more',
+		if ( controls.more.length ) {
+			controls.main.push( {
+				text: translate( 'More' ),
+				className: 'more',
 				onClick: this.props.onShowMore,
-				icon: 'ellipsis'
+				icon: 'ellipsis',
 			} );
 
-			extraControls.push( {
-				text: this.translate( 'Back' ),
-				className: 'post-controls__back',
+			controls.more.push( {
+				text: translate( 'Back' ),
+				className: 'back',
 				onClick: this.props.onHideMore,
-				icon: 'chevron-left'
+				icon: 'chevron-left',
 			} );
+		}
 
-			mainControls = this.buildControls( availableControls );
-			moreControls = this.buildControls( extraControls );
-			postControlsClass += ( extraControls.length <= 2 ) ? ' post-controls--desk-nomore' : '';
+		return controls;
+	}
 
-			return (
-				<div className={ postControlsClass }>
-					<ul className="post-controls__pane post-controls__more-options">
-						{ moreControls }
-					</ul>
-					<ul className="post-controls__pane post-controls__main-options">
-						{ mainControls }
-					</ul>
-				</div>
-			);
-		} else {
-			mainControls = this.buildControls( availableControls );
+	render() {
+		const controls = this.setControls();
+		let postControlsClass = 'post-controls';
```

Comment:
>I've noticed that `post-controls--desk-nomore` isn't used anywhere else in Calypso.
So I thought I could just remove it

Hmm, yeah, if it's not in the codebase, probably safe to remove. Based on the name, perhaps something from desktop app? I don't see anything in [that codebase](https://github.com/Automattic/wp-desktop) either (cc @sendhil in case this rings any bells).